src/request/create/ChooseIdenticon.js

```diff
@@ -0,0 +1,127 @@
+class ChooseIdenticon extends Nimiq.Observable {
+    /**
+     * @param {EncryptionType} encryptionType
+     * @param {HTMLElement} [$el]
+     */
+    constructor(encryptionType, $el) {
```

Comment:
I understand that the encryption type is necessary for instantiating a `Key`, but it's rather unneeded in the identicon selector, no? Can't the selector return a `KeyPair`, instead of a `Key` instance?